benchmark/test_biocypher_query_generation.py

```diff
@@ -11,19 +13,31 @@
     None,
 )
 
-MODEL_NAMES = [
+OPENAI_MODEL_NAMES = [
     "gpt-3.5-turbo",
     "gpt-4",
 ]
 
+XINFERENCE_MODEL_NAMES = [
```

Comment:
I added these as a placeholder to figure out how we will manage starting, stopping, and calling different models from within the pytest process. Selecting these does not currently work, so running `pytest benchmark` will give errors for these parts of the matrix.

Reply:
Sounds good!